feat: add Oracle Cloud-specific tags to hosts running on OCI (#21957)
newrelic/docs-website: src/content/docs/release-notes/infrastructure-release-notes/infrastructure-agent-release-notes/new-relic-infrastructure-agent-1700.mdx

---
subject: Infrastructure agent
releaseDate: '2025-10-21'
version: 1.70.0
---
A new version of the agent has been released. Follow standard procedures to [update the Infrastructure agent](https://docs.newrelic.com/docs/infrastructure/install-configure-manage-infrastructure/update-or-uninstall/update-infrastructure-agent).
New Relic recommends that you upgrade the agent regularly and at a minimum every 3 months. As of this release, the oldest supported version is [Infrastructure agent 1.58.0](https://docs.newrelic.com/docs/release-notes/infrastructure-release-notes/infrastructure-agent-release-notes/new-relic-infrastructure-agent-1580/).

## Changed
* feat(oci): add Oracle Cloud-specific tags to hosts running on OCI in https://github.com/newrelic/infrastructure-agent/pull/2121

>   Starting with version 1.70.0, the new relic infrastructure agent collects the following details for VM’s running inside Oracle Cloud Infrastructure (OCI): 

Reviewer: ```suggestion
  Starting with version 1.70.0, the New Relic Infrastructure agent now automatically collects metadata for VMs running in Oracle Cloud Infrastructure (OCI). The agent collects the following details:
```

  - displayName
  - host.id
  - oci.availabilityDomain
  - oci.region
  - oci.compartmentId
  - oci.shape
  - oci.imageId
  

>   These details are available as tags for the New Relic entity search, and as attributes for queries via NRQL. Additionally, the host.id is important for the upcoming NR Cloud Integration for OCI, to make sure that any additional metadata will get attached to the correct Host entity.

Reviewer: ```suggestion
  You can use this metadata as tags for entity search or as attributes in NRQL queries. The `host.id` attribute is also used by the New Relic Cloud Integration for OCI to attach cloud metadata to the correct host entity.
```

>   The infra agent metadata functionality is enabled by default. Customers who would like to disable this feature can use the configuration items [disable_cloud_metadata](https://docs.newrelic.com/docs/infrastructure/infrastructure-agent/configuration/infrastructure-agent-configuration-settings/#disable-cloud-metadata) or [disable_cloud_instance_id](https://docs.newrelic.com/docs/infrastructure/infrastructure-agent/configuration/infrastructure-agent-configuration-settings/#disable-cloud-instance-id).

Reviewer: ```suggestion
 This feature is enabled by default to ensure consistency with existing metadata. If you need to disable this feature, you can use the configuration settings [disable_cloud_metadata](https://docs.newrelic.com/docs/infrastructure/infrastructure-agent/configuration/infrastructure-agent-configuration-settings/#disable-cloud-metadata) or [disable_cloud_instance_id](https://docs.newrelic.com/docs/infrastructure/infrastructure-agent/configuration/infrastructure-agent-configuration-settings/#disable-cloud-instance-id).
```

Reviewer: > "to ensure consistency **with existing metadata**" - this sounds to me like a different meaning. Unless you meant the metadata on other cloud providers.

Thanks for your feedback @nr-ksteinbach. Let's rephrase it to: 

```suggestion
  The infrastructure agent metadata functionality is enabled by default. You can disable this feature using the configuration settings [disable_cloud_metadata](https://docs.newrelic.com/docs/infrastructure/infrastructure-agent/configuration/infrastructure-agent-configuration-settings/#disable-cloud-metadata) or [disable_cloud_instance_id](https://docs.newrelic.com/docs/infrastructure/infrastructure-agent/configuration/infrastructure-agent-configuration-settings/#disable-cloud-instance-id).
```
